data-config.json

```diff
@@ -19,6 +20,7 @@
         "html": true,
         "html-content": false,
         "slingmodel": true,
+        "testclass": true,
```

Comment:
Since the result of this is a class of unfinished tests, which are not very easy to write, let's default this option to `false`